builders/manage/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Manage your ContainerChain
+description: After you deploy your ContainerChain, you can maintain and upgrade your ContainerChain with these guides, including actions such as using the Sudo account to take privileged actions.
```

Comment:
the description should be 160 characters or less. can you please shorten this one and check the others to make sure they're between 120-160 characters ?